.github/workflows/test.yaml

```diff
@@ -77,10 +77,10 @@ jobs:
         # test latest features and compatibility of lower version
         include:
           - feature_version: 6.0.0
-            tidb_version: nightly
+            tidb_version: latest
```

Comment:
latest and nightly are different. latest should be pointing to stable release only I guess.

Reply:
How about introducing both latest and nightly to our matrix?

IMO currently the source of unstable comes from our tests, instead of nightly.

Reply:
Make sense.

I found some unstable cases when restarting tiup playground nightly in m1. It can be resolved by the cypress default env variable.